AmericanMedicalAssociation/AMA-style-guide
# EWL-3572: adds the course browser tabs organism.

**Jira Ticket**

- [EWL-3572: EC | Design Browse Tabs component](https://issues.ama-assn.org/browse/EWL-3572)


## Description

Creates the course browser tabs for the Education Center section


## To Test

- [ ] Navigate to Organisms > EC > Ec Course Browser Tabs
- [ ] Verify that the Education Center Course Browser meets [requirements](https://docs.google.com/document/d/11SpEl1T1uRZ2HPg2g6i-RprZhsPXo_pt2W3xMbCnNZo/edit#):

> This section will allow for 2-5 tab groups. Each group will have 3-4 Course Cards and a Call to 
> Action link. The primary call to action will remain static throughout all of the tabs.
Fields:
Section Title (text)
Tab group (allows 2-5 items, implemented using Quick Tabs)
Group Label (text)
Course cards (allows 3-4 cards)
Primary CTA (link, default text: “Start a Course”, default url: “https://searchec.ama-assn.org”)
Secondary CTA (link field, custom text and url)
Link will change depending on which tab is currently selected, for example “Browse Patient Care Courses” should link to Patient Care courses.



## Relevant Screenshots/GIFs

N/A


## Remaining Tasks

N/A


## Additional Notes

Dynamic aspects to be included in Drupal implementation of tabs.
